### PR TITLE
Fix2

### PR DESCRIPTION
**Pull request by lw453**
This PR adds further fixes to the RMC break search code, most importantly missing freeze arguments. It also adds configs for requester-pays to the batch script and removes references to specific users (e.g. directs the batch temp directory to a more general bucket). Minor change: removal of duplicate characters in `lstrip` calls.

### CODE REVIEW

**Review by ch-kr — COMMENTED** [review]
quick comment/question, otherwise LGTM

File: rmc/pipeline/two_breaks/run_batches.py [hunk 1]
```diff
@@ -821,7 +827,7 @@ def main(args):
     parser.add_argument(
         "--batch-bucket",
         help="Bucket provided to hail batch for temporary storage.",
-        default="gs://gnomad-tmp/kc/",
+        default=TEMP_PATH_WITH_FAST_DEL,
```
[the remark this answers is not in the document]
Comment (lw453) [reply rmc/pipeline/two_breaks/run_batches.py:830]: Thanks for adding this!

File: rmc/pipeline/two_breaks/run_batches.py [hunk 2]
```diff
@@ -442,7 +442,7 @@ def process_section_group(
     hl.init(
         spark_conf={
             "spark.hadoop.fs.gs.requester.pays.mode": "CUSTOM",
-            "spark.hadoop.fs.gs.requester.pays.buckets": f"{requester_pays_bucket.lstrip('gs://')}",
+            "spark.hadoop.fs.gs.requester.pays.buckets": f"{requester_pays_bucket.lstrip('gs:/')}",
```
[the remark this answers is not in the document]
Comment (ch-kr) [reply rmc/pipeline/two_breaks/run_batches.py:445]: oooh yes. thanks!

**Review by ch-kr — APPROVED** [review]
LGTM!

**Pull request merged**